src/m2n/DistributedCommunication.hpp

```diff
@@ -117,6 +120,33 @@ class DistributedCommunication
    */
   virtual void broadcastReceive(double &itemToReceive) = 0;
 
+  /**
+   * @brief All ranks send their mesh partition to remote local  connected ranks.
+   */
+  virtual void broadcastSendMesh() = 0;
+  
+  /**
+   * @brief All ranks receive mesh partition from remote local ranks.
+   */
+  virtual void broadcastReceiveMesh() = 0;
+
+  /*
+   * A mapping from remote local ranks to the IDs that must be communicated
+   */
+  using communicationMap = std::map<int, std::vector<int>>;
```

Comment:
Types (names and aliases) are capitalised.

```suggestion
  using CommunicationMap = std::map<int, std::vector<int>>;
```

Reply:
@fsimonis Yes, right. changed!